sys-devel/llvm/llvm15-15.0.7.recipe

```diff
@@ -0,0 +1,739 @@
+		libclangParse \
+		libclangPseudo \
+		libclangPseudoCLI \
+		libclangPseudoCXX \
+		libclangPseudoGrammar \
+		libclangQuery \
+		libclangReorderFields \
+		libclangRewrite \
+		libclangRewriteFrontend \
+		libclangSema \
+		libclangSerialization \
+		libclangStaticAnalyzerCheckers \
+		libclangStaticAnalyzerCore \
+		libclangStaticAnalyzerFrontend \
+		libclangSupport \
+		libclangTidy \
+		libclangTidyAbseilModule \
+		libclangTidyAlteraModule \
+		libclangTidyAndroidModule \
+		libclangTidyBoostModule \
+		libclangTidyBugproneModule \
+		libclangTidyCERTModule \
+		libclangTidyConcurrencyModule \
+		libclangTidyCppCoreGuidelinesModule \
+		libclangTidyDarwinModule \
+		libclangTidyFuchsiaModule \
+		libclangTidyGoogleModule \
+		libclangTidyHICPPModule \
+		libclangTidyLinuxKernelModule \
+		libclangTidyLLVMLibcModule \
+		libclangTidyLLVMModule \
+		libclangTidyMain \
+		libclangTidyMiscModule \
+		libclangTidyModernizeModule \
+		libclangTidyMPIModule \
+		libclangTidyObjCModule \
+		libclangTidyOpenMPModule \
+		libclangTidyPerformanceModule \
+		libclangTidyPlugin \
+		libclangTidyPortabilityModule \
+		libclangTidyReadabilityModule \
+		libclangTidyUtils \
+		libclangTidyZirconModule \
+		libclangTooling \
+		libclangToolingASTDiff \
+		libclangToolingCore \
+		libclangToolingInclusions \
+		libclangToolingRefactoring \
+		libclangToolingSyntax \
+		libclangTransformer \
+		libclangdRemoteIndex \
+		libclangdSupport \
+		libfindAllSymbols \
+		libLLVM* \
+		libLTO	\
+		liblld* \
+		libRemarks
+
+	# adjust cmake library paths because prepareInstalledDevelLibs moved them
+	sed -i 's,\${_IMPORT_PREFIX}/lib/,\${_IMPORT_PREFIX}/develop/lib/,' \
+		$libDir/cmake/clang/ClangTargets-release.cmake \
+		$libDir/cmake/lld/LLDTargets-release.cmake \
+		$libDir/cmake/llvm/LLVMExports-release.cmake
+
+	sed -i 's|/libexec/|/lib/|' $binDir/scan-build
+	sed -i 's|/share/|/data/|' $binDir/scan-build $binDir/scan-view
+
+	# clang package
+	packageEntries clang \
+		$binDir/c-index-test \
+		$binDir/clang* \
+		$binDir/git-clang-format \
+		$binDir/modularize \
+		$dataDir/clang \
+		$includeDir/clang* \
+		$libDir/clang \
+		$libDir/cmake/clang \
+		$developLibDir/libclang*
+
+	# analysis package
+	packageEntries clang_analysis \
+		$binDir/scan-build \
+		$binDir/scan-view \
+		$libDir/c++-analyzer \
+		$libDir/ccc-analyzer \
+		$dataDir/scan-build \
+		$dataDir/scan-view \
+		$manDir/man1/scan-build.1
+
+	# lld package
+	packageEntries lld \
+		$binDir/ld.lld \
+		$binDir/ld64.lld \
+		$binDir/lld \
+		$binDir/lld-link \
+		$binDir/wasm-ld \
+		$includeDir/lld* \
+		$developLibDir/liblld*
+
+	# libs package
+	packageEntries libs \
+		$libDir/libclang* \
+		$libDir/libLLVM* \
+		$libDir/libLTO* \
+		$libDir/libRemarks*
+
+	# python3 package
+	packageEntries python3 \
+		$binDir/lit \
+		$prefix/lib/python*
+}
+
+TEST()
+{
+	export LIT_ARGS="-v"
+	if [ "$targetArchitecture" = x86_gcc2 ];then
+		export LIBRARY_PATH=$LIBRARY_PATH:$sourceDir/build/lib/x86
```

Comment:
this could use `$relativeLibDir` at the end, replacing the whole if, or `$secondaryArchSubDir`

Reply:
Sorry for hijacking this PR, but any chance that one of you guys will add support for and enable the sanitizers either in llvm-15 or gcc-12?